subprojects/packagefiles/fio/meson.build

```diff
@@ -0,0 +1,37 @@
+project(
+  'fio',
+  'c',
+  version: '3.28',
+)
+fs = import('fs')
+
+if not fs.exists('config-host.h')
+  message('Configuring ..')
+  run_command('./configure', capture: false, check: true)
+endif
+if not fs.exists('fio')
+  cpu_count = run_command([
+    import('python').find_installation('python3'),
```

Comment:
I think here you don't have to look for python again. You can just use py.

Reply:
I think this is needed, subprojects are technically stand-alone. The meson build file is only here because the actual subproject (fio) doesn't provide one itself. I don't think they re-use scope and definitions, but we can test that.

Reply:
Subprojects are sandboxed and variables from one are not defined in another. You need to look it up again.

However, please don't use `import('python').find_installation('python3')` just to run a script. Found installations are identical to `find_program('python3')` except that they also run additional introspection commands to scrape the python interpreter for information useful for the primary point of the python module: compiling python C-API extensions, and/or installing modules to the python site-packages.

Doing this for nothing other than running a script accomplishes nothing that `find_program('python3')` doesn't already do, other than wasting CPU cycles to import `mesonbuild.modules.python` and run a json dumper for distutils/sysconfig internal state.

Reply:
> Subprojects are sandboxed and variables from one are not defined in another. You need to look it up again.
Noted. Had not used subprojects before. Thx for the feedback.

> please don't use import('python').find_installation('python3') just to run a script. Found installations are identical to find_program('python3')...
So, Unless we are using the meson's python module, it does not make sense to use import('python').find_isntallation. right?

Reply:
wrt. `find_program('python3')` - changed, thanks.